externalId-migration-util/src/main/java/org/sunbird/migration/RecordProcessor.java

```diff
@@ -323,14 +330,18 @@ private boolean performSequentialOperationOnRecord(
       UserDeclareEntity userDeclareEntity, Map<String, String> orgProviderMap) {
 
     try {
+      String provider = orgProviderMap.get(userDeclareEntity.getProvider());
+      if (null == provider) {
```

Comment:
pls include a log statement on this. and a similarly in the place above (in another file as well).